frontend/tests/util/isolation.ts

```diff
@@ -94,11 +94,20 @@ export const test = base.extend<CustomTestFixtures, CustomWorkerFixtures>({
 });
 
 const runTobiraCommand = async (tobira: TobiraProcess, args: string[]) => {
-    await new Promise(resolve => {
+    await new Promise((resolve, reject) => {
         args.push("-c");
         args.push(tobira.configPath);
         const p = childProcess.spawn(tobira.binaryPath, args);
-        p.on("close", resolve);
+        p.on("close", code => {
+            if (code === 0) {
+                resolve(undefined);
+            } else {
+                const cmd = "tobira " + args.slice(0, -2)
+                    .map(arg => arg.includes(" ") ? `"${arg}"` : arg)
+                    .join(" ");
+                reject(new Error(`Tobira process (${cmd}) ended with failure, code = ${code}`));
+            }
+        });
     });
 };
```

Comment:
Most of this could be boiled down to using `promisify` from `node:util` on `spawn`.

Reply:
[This](https://stackoverflow.com/q/72862197/2408867) and [this](https://github.com/barthofu/spawnise) seem to imply that using `promisify` with `spawn` is not possible or at least not straight forward. And indeed, trying it naively doesn't work. So I would probably keep it like that? It's not that bad...

Reply:
Apparently I was thinking `exec`. Which I guess you can't use here? Feel free to resolve if not. 🤷‍♀️